### ISSUE TITLE
Need baseline classification metrics using a vanilla neural network

### ISSUE THREAD

**Issue by zhao1701** [zhao1701/supervised-vae#8]
We should be able to make a statement as to whether using our new architecture helps classification performance or hurts it. And if the architecture hurts classification performance, we need to decide if the penalty is small enough to still justify trying to visualize the latent dimensions.

I think someone should throw together a simple model in Keras using the same classifier architecture we've built into the SVAE (minus the variational stuff) and just assess how a regular neural network does. The metrics we should collect are ** test accuracy**, **test AUROC**, and **test average precision**.
